PhysicsTools/PatAlgos/plugins/PATTriggerProducer.cc

```diff
@@ -558,7 +558,7 @@ void PATTriggerProducer::produce(Event& iEvent, const EventSetup& iSetup) {
       if (hltConfig.prescaleSize() > 0) {
         if (hltPrescaleProvider_.prescaleSet(iEvent, iSetup) != -1) {
```

Comment:
Is this ok with rounding effects?

Reply:
`prescaleSet` is the index of the PS column, and it remains an `int`, so no issue.